precompiles/avs/avs.sol

```diff
@@ -14,11 +14,20 @@ IAVSManager constant AVSMANAGER_CONTRACT = IAVSManager(
 /// @custom:address 0x0000000000000000000000000000000000000902
 interface IAVSManager {
     function AVSAction(
+        string[] memory avsOwnerAddress,
         string memory avsName,
-        string memory avsAddress,
-        string memory operatorAddress,
+        string memory slashContractAddr,
+        string[] memory assetID,
         uint64 action,
-        string memory avsOwnerAddress,
-        string memory assetID
+        uint64 minimumDelegation,
+        uint64 unbondingEpochs
```

Comment:
unbondingTime/unbondingPeriod would be better.

Reply:
fixed